content/docs/atom-guide/look-dev/six-point-lighting-tutorial.md

```diff
@@ -0,0 +1,439 @@
+         {{< /note >}}
+    * **Opacity**
+      * **Opacity Mode**: `Blended`
+      * **Alpha Source**: `Packed`
+         {{< note >}}
+         A `Packed` alpha source means the material will use the alpha channel packed into the base color texture.
+         {{< /note >}}
+      * **Factor**: `1.0`
+      * **Alpha affects specular**: `1.0`
+    * **UVs**
+      * **Center** > **U**: `0.0`
+      * **Center** > **V**: `0.0`
+    * **General Settings**
+      * **Double-sided**: Enabled
+         {{< note >}}
+         Enabling this setting allows for rendering of the back side of the material.
+         {{< /note >}}
+   
+6.  In the **Editor**, make an entity with **Mesh** and **Material** components. Choose a plane for the **Mesh** (`o3de/Gems/Atom/Tools/MaterialEditor/Assets/MaterialEditor/ViewportModels/Plane_1x1.fbx`) and the material you just created for the material.
+
+{{< image-width src="/images/atom-guide/six-point-lighting/material.png" width="100%" alt="Material added." >}}
+
+As of now, the entity should just display the whole alpha texture with all the frames.
+
+{{< image-width src="/images/atom-guide/six-point-lighting/all-frames.png" width="100%" alt="All frames of the texture." >}}
+
+## Add animation
+The next step is to add animation to the material. The textures contain all the frames of the animation so you will programatically iterate through the frames.
+
+1. Open `SixPointLighting_Common.azsli`.
+   
+2. At the bottom, add a function to get the position of the correct frame in the texture map according to the time.
+   ```glsl
+   float2 GetUvForCurrentFrame(float2 baseUv)
+   {
+      // Fixed frequency of 30hz
+      // Get the current frame
+      float frame = (float)(((double)SceneSrg::m_time / (33.3333)) * 1000.0) % (MaterialSrg::m_columnCount * MaterialSrg::m_rowCount);
+
+      if(o_enableDebugFrame)
+      {
+         // The frame input by the material is 1-indexed, so subtract 1 here to make it 0-indexed
+         frame = MaterialSrg::m_debugFrame - 1.0f;
+      }
+
+      // Get the row/column of the frame
+      float frameColumn = floor(frame % MaterialSrg::m_columnCount);
+      float frameRow = floor(frame / MaterialSrg::m_columnCount) % MaterialSrg::m_rowCount;
+      
+      float2 invColumnRowCounts = float2(1.0f, 1.0f) / float2(MaterialSrg::m_columnCount, MaterialSrg::m_rowCount);
+      float2 sixPointUv = (baseUv + float2(frameColumn, frameRow)) * invColumnRowCounts;
+
+      return sixPointUv;
+   }
+   ```
+   {{< note >}}
+   This function includes an if statement for if you enabled debugging for a specific frame in the **Material Editor**. If so, then the frame will be the frame specified instead of the frame according to the time. This functionality can be helpful for looking at specific frames of the animation to ensure lighting is being applied correctly.
```

Comment:
Attempted to rephrase the first sentence so it flows better.
```suggestion
   The condition, `if(o_enableDebugFrame)`, occurs if you enabled debugging for a specific frame, which can be set via the **Material Editor**. If enabled, this function uses the specified frame instead of the current frame. This functionality can help ensure that lighting is correctly applied in a specific frame.  
```